components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -455,20 +469,24 @@ void JsonRpcService::GetAllNetworks(GetAllNetworksCallback callback) {
   std::move(callback).Run(std::move(all_chains));
 }
 
-std::string JsonRpcService::GetNetworkUrl() const {
-  return network_url_.spec();
+std::string JsonRpcService::GetNetworkUrl(mojom::CoinType coin) const {
+  if (!network_urls_.contains(coin))
+    return std::string();
+  return network_urls_.at(coin).spec();
```

Comment:
DITTO, how about `return network_urls_.contains(coin) ? network_urls_[coin].spec() : "";`?

Reply:
fixed in https://github.com/brave/brave-core/pull/12461/commits/cea463abc30fc02b69d035171fd3525822495657